flask-service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: todo-app-service
+spec:
+  type: NodePort
```

Comment:
what does this type NodePort imply?

Reply:
externally service ah access pantrathuku Nodeport use pantrom. internal access only we want to use cluster type service.